config/crd/bases/flows.netobserv.io_flowcollectors.yaml

```diff
@@ -1353,6 +1353,22 @@ spec:
                     minimum: 0
                     type: integer
                 type: object
+              kafka:
+                description: Kafka configurations, if empty the operator will deploy
+                  a all-in-one FLP
+                properties:
+                  address:
+                    default: ""
+                    description: Address of the kafka server
+                    type: string
+                  topic:
+                    default: ""
+                    description: Kafka topic to use
+                    type: string
+                required:
+                - address
+                - topic
+                type: object
```

Comment:
For the area of using FLP as ingress ... we will have multiple FLP instances consuming from the same topic. Need to make sure that we have some configuration on the distribution that Kafka is creating when sending data into those FLP's going forward this is going to be very important after we implement the stateful connection tracking capabilities. There might be multiple different configurations based on Load/ number of consumers etc ... those might need to be exported and configurable to users.

Reply:
Yes, the kafka configuration block is meant to change, but I would prefer to change it again once we have the connection tracking so we can test the different configurations and chose what we need to expose.
Would you be fine with that approach?

Reply:
Yes, it doesn't have to be in this PR, agree ... maybe just open a ticket somewhere to remember that we need to improve that later.